### PR TITLE
Tweak: Kit-Library - Clicking a page in the kit overview opens live preview [ED-3134]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

**Review by iNewLegend — APPROVED** [review]

**Comment by iNewLegend** [comment]
@Nevoss Screenshoer fails:
```
>> Error: File to import not found or unreadable: ../../modules/kit-library/assets/js/pages/overview/overview-content.scss.
>>         on line 51 of core/app/assets/styles/app-imports.scss
>>         from line 11 of assets/dev/scss/app/app.scss
>>         from line 5 of assets/dev/scss/direction/app.scss
>> >> @import "../../modules/kit-library/assets/js/pages/overview/overview-content
>>    ^
```

**Comment by Nevoss** [comment]
@iNewLegend Fixed

File: core/app/modules/kit-library/assets/js/pages/overview/overview-content-group-item.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Document from '../../models/document';
-import { Card, CardBody, CardHeader, CardImage, Heading } from '@elementor/app-ui';
+import { Card, CardBody, CardOverlay, CardHeader, CardImage, Heading, Button } from '@elementor/app-ui';
```
Comment (matipojo) [inline core/app/modules/kit-library/assets/js/pages/overview/overview-content-group-item.js:2]: a-z

File: core/app/modules/kit-library/assets/js/pages/overview/overview.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import Content from 'elementor/core/app/assets/js/layout/content';
-import OverviewContent from './overview-content';
 import ItemHeader from '../../components/item-header';
 import OverviewSidebar from './overview-sidebar';
 import Layout from '../../components/layout';
 import useKit from '../../hooks/use-kit';
 import useKitDocumentByType from '../../hooks/use-kit-document-by-type';
 import { useMemo } from 'react';
 import { useNavigate } from '@reach/router';
+import OverviewContentGroup from './overview-content-group';
```
Comment (matipojo) [inline core/app/modules/kit-library/assets/js/pages/overview/overview.js:9]: a-z

File: core/app/modules/kit-library/assets/js/pages/preview/preview.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import Layout from '../../components/layout';
 import PreviewResponsiveControls from './preview-responsive-controls';
 import useKit from '../../hooks/use-kit';
 import { PreviewIframe } from './preview-iframe';
-import { useNavigate } from '@reach/router';
+import { useNavigate, useLocation } from '@reach/router';
```
Comment (matipojo) [inline core/app/modules/kit-library/assets/js/pages/preview/preview.js:6]: a-z

File: core/app/assets/styles/app-imports.scss [hunk 4]
```diff
@@ -48,6 +48,6 @@
 @import "../../modules/kit-library/assets/js/pages/index/index-header.scss";
 @import "../../modules/kit-library/assets/js/pages/preview/preview-responsive-controls.scss";
 @import "../../modules/kit-library/assets/js/pages/preview/preview-iframe.scss";
-@import "../../modules/kit-library/assets/js/pages/overview/overview-content.scss";
 @import "../../modules/kit-library/assets/js/pages/overview/overview-sidebar.scss";
 @import "../../modules/kit-library/assets/js/components/favorites-actions.scss";
+@import "../../modules/kit-library/assets/js/pages/overview/overview.scss";
```
Comment (matipojo) [inline core/app/assets/styles/app-imports.scss:53]: a-z

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**